Library/AppEnvironment.swift

```diff
@@ -295,6 +296,41 @@ public struct AppEnvironment: AppEnvironmentType {
     )
   }
 
+  internal static func accountNameForUserId(_ userId: Int) -> String {
+    return "kickstarter_\(userId)"
```

Comment:
This is something I went back-and-forth on. The keychain is designed to store passwords for accounts. However, the way our app works right now, we only ever store one OAuth token in the user defaults - we store it for the current user, and remove it when you log out. 

So I had two choices here. I could name the keychain account something like `kickstarter_current_user` and only have one, or I could name it after the _specific_ user who is logged in. 

I decided to keep it linked to the specific user, but there's an argument to be made that YAGNI and I'm over-complicating things.

Reply:
Yeah, I'm not sure I see a need any more, either. My original thinking was "hey, this way we can support multiple users in the future!" but I'm starting to think that's not ... particularly useful. You could definitely talk me into making it a single key. 

As for the naming scheme, great question. We do [use the bundle ID in Keychain.swift](https://github.com/kickstarter/ios-oss/blob/main/Library/Keychain.swift#L21), which should identify it uniquely, but I haven't actually tested to make sure that it doesn't overlap between build environments. I think you have to do _extra_ work to enable keychain sharing between app IDs but yeah, let's validate that.

Reply:
Double-checked and yes, we're already using the bundle ID when we save the key. So there won't be any overlap.